### PR TITLE
Reporter Updates

### PR DESCRIPTION
**Pull request by flooie**
PR tweaks and updates reporters.json

Specifically, updating the Southern and Southwestern Reporter variations and regex patterns
Adding the various LEXIS reporters
Add a few one off updates to small various edge cases

### CODE REVIEW

**Comment by flooie** [comment]
To the reviewer - it looks like the rebase merger duplicated the commits.  Apologies.

File: reporters_db/data/reporters.json [hunk 1]
```diff
@@ -14184,13 +16766,28 @@
...
+                        "$volume S\\.?-? ?(W|w)\\.?,? ?\\(?2n?d,?\\.?\\)?( Series,?)? $page",
+                        "$volume $reporter (?P<page>[LXIV]+);?"
+                    ],
                     "start": "1928-01-01T00:00:00"
                 },
                 "S.W.3d": {
                     "end": null,
+                    "regexes": [
+                        "$full_cite",
+                        "$volume $reporter (?P<page>[LXIV]+);?"
```
Comment (flooie) [inline reporters_db/data/reporters.json:16780]: @mlissner sw.3d roman numerals

File: reporters_db/data/reporters.json [hunk 2]
```diff
@@ -14184,13 +16766,28 @@
                 },
                 "S.W.2d": {
                     "end": "1999-12-31T00:00:00",
+                    "regexes": [
+                        "$full_cite",
+                        "$volume S\\.?-? ?(W|w)\\.?,? ?\\(?2n?d,?\\.?\\)?( Series,?)? $page",
+                        "$volume $reporter (?P<page>[LXIV]+);?"
```
Comment (flooie) [inline reporters_db/data/reporters.json:16772]: @mlissner sw.2d roman numerals

File: reporters_db/data/reporters.json [hunk 3]
```diff
@@ -14640,9 +17263,17 @@
                 },
                 "So. 3d": {
                     "end": null,
+                    "regexes": [
+                        "$full_cite",
+                        "$volume $reporter (?P<page>[LXIV]+);?"
```
Comment (flooie) [inline reporters_db/data/reporters.json:17268]: So.3d Roman numerals

**Comment by mlissner** [comment]
This all looks fine to me, though putting a more sophisticated regex for roman numerals seems like a decent idea. 

@jcushman, no need to look at this, but in case you're interested I thought I'd flag you. Reporters DB grows again.

**Comment by mlissner** [comment]
I'm merging so you can keep up your velocity, but @jcushman if you find a minute and the interest, we can revert if you catch something.

**Pull request merged**